Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -559,6 +559,130 @@ final class WorkspaceTests: XCTestCase {
             }
         }
     }
+    
+    func testPackageEditAndRemove() {
```

Comment:
We basically want to check if we can unedit the dependencies which were removed. So I'd call this `testCanUneditRemovedDependencies()` or similar.